db/etl/migrate/20200715110603_add_accepts_priority_pushed_cases_index_on_organization.rb

```diff
@@ -0,0 +1,5 @@
+class AddAcceptsPriorityPushedCasesIndexOnOrganization < Caseflow::Migration
+  def change
+    add_safe_index :organizations, :accepts_priority_pushed_cases, algorithm: :concurrently
```

Comment:
```suggestion
    add_safe_index :organizations, :accepts_priority_pushed_cases
```
You can leave `algorithm` out due to https://github.com/department-of-veterans-affairs/caseflow/blob/e171e539bbe5673e3bee679d377f5304d1d06302/lib/caseflow/migration.rb#L18